containers/systemd-service

```diff
@@ -59,13 +57,22 @@ function run {
   fi
 }
 
+function sdproxy() {
+  SD_NOTIFY_PROXY=/usr/local/pf/bin/sdnotify-proxy
+  SD_SOCK=/tmp/$1.sock
+  CR_SOCK=/var/run/$1.sock
+  WATCHDOG_USEC_D=10000000
+  echo "SDPROXY cli: $SD_NOTIFY_PROXY $SD_SOCK /usr/bin/docker run --volume=$SD_SOCK:$CR_SOCK --env NOTIFY_SOCKET=$CR_SOCK --env WATCHDOG_USEC=$WATCHDOG_USEC_D ${@:2}"
```

Comment:
I'd remove this line here since keeping this line and the one below may lead them to be out of sync

Reply:
Done